src/extend.js

```diff
@@ -16,11 +16,11 @@ limitations under the License.
 
 'use strict';
 
-module.exports = function(dest, src) {
+export default function(dest, src) {
```

Comment:
I thought `export default` broke things?

Reply:
It does for the js-sdk, not for the react-sdk. We don't have a central index for the react-sdk that we need to worry about